org.lflang/src/org/lflang/generator/c/CGenerator.java

```diff
@@ -1622,6 +1622,12 @@ private void recordBuiltinTriggers(ReactorInstance instance) {
         }
     }
 
+    private void assignLetToReaction(ReactorInstance instance) {
+        for (ReactionInstance reaction : instance.reactions) {
```

Comment:
Where does the return value of `getLogicalExecutionTime ` go? If we are not going to use the return value, shouldn't the return type void?

Reply:
Yes, this function was written for testing. So, I think it should be deleted.